model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -1,23 +1,9 @@
-
 def format_import_proto_library(name, src, deps):
-  """This macro exists as a way to build proto files that contain import statements in both Gradle
+  """
+  This macro exists as a way to build proto files that contain import statements in both Gradle
   and Bazel.
-  Currently the files that use this macro are: exploration.proto, topic.proto, and question.proto.
   This macro formats the src file's import statements to contain a full path to the file in order
   for Bazel to properly locate file.
-  The genrule() rule generates an altered copy of the src file for Bazel to build, leaving the
-  unaltered copy for Gradle to use.
-  The proto_library() rule takes this altered copy and builds it into a proto_library that a
-  java_lite_proto_library() rule can depend on.
-
-  Args:
```

Comment:
Keep these args--they're still useful for describing _how_ to use the macro. My earlier comment was that the bit above this that you removed was basically just describing the implementation. The argument portions of the documentation are important though--without them callers might not use the macro correctly.

Reply:
Okay!